### PR TITLE
Add OpenBSD support for fs subpackage

### PR DESCRIPTION
**Pull request by nogoegst**

### CODE REVIEW

File: fs/stat_openbsd.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// +build openbsd
```
Comment (dnephin) [inline fs/stat_openbsd.go]: Isn't this identical to `stat_linux.go` ? could we use the same file with build tags?

Comment (nogoegst) [reply fs/stat_openbsd.go]: oh wow, nice catch!
`stat_linux.go` has to be renamed though. what name should it have?

**Comment by nogoegst** [comment]
hey there again!
I made corresponding changes and squashed all the things :)

**Comment by nogoegst** [comment]
gentle ping :)

**Comment by nogoegst** [comment]
@dnephin ping :)

**Comment by nogoegst** [comment]
I have no idea why CI failed here in one case.

**Comment by nogoegst** [comment]
It seems it was a CI issue last time.

**Pull request merged**